jbpm-db-scripts/src/main/resources/db/upgrade-scripts/sybase/jbpm-7.60-to-7.61.sql

```diff
@@ -0,0 +1 @@
+alter table TaskEvent add column currentOwner varchar(255)
```

Comment:
this change is missing for the create sybase schema `sybase-jbpm-schema.sql` - need to be added there